### ISSUE TITLE
MQTT sendet keine Werte für BAT_DATA/1/BAT_DCB................

### ISSUE THREAD

**Issue by smartboart** [rxhan/RSCPGui#11]
BAT_DATA/1/BAT_DATA/BAT_DCB......... States werden angelegt aber es werden keine Daten übermittelt.
(Batterie Strang 2 ) vergleichbar mit den werten aus der  gui   BAT #1 DCB #0 wo die Werte angezeigt werden.
testin3 Branch auf linux Debian Buster

**Comment by rxhan** [comment]
Werden denn die Werte in dem Exportfenster in der Vorschau richtig angezeigt? Habe leider keinen zweiten Bat-Strang, deswegen fällt es mir etwas schwer das nachzustellen.

**Comment by rxhan** [comment]
Ich seh grad, die Formatierung diverser Felder sieht ja mies aus, dass werd ich noch beheben.

**Comment by smartboart** [comment]
Ja, die Messwerte werden richtig in der vorschau angezeigt, sie werden nur nicht per Mqtt übertragen. Jedoch werden die States angelegt aber nicht gefüllt.

**Comment by rxhan** [comment]
Du kannst mit -v DEBUG -f die Logausgaben erhöhen und in eine Datei schreiben lassen. Du könntest mir die Datei zukommenlassen. Darin sind dann jedoch in dem Loglevel auch Zugangsdaten enthalten, du solltest also aus der Logdatei den Verbindungsversuch nach Programmstart herauslöschen. Vielleicht kann ich damit das Problem besser nachvollziehen.

**Comment by smartboart** [comment]
Wow...super...du warst ja richtig fleissig über Ostern... Habe mitr die testin3 gezogen und werde testen. Werde -v -f aktivieren und dir die log Datein dann zu senden.
